pkg/prompts/prompts.go

```diff
@@ -885,11 +885,11 @@ func contains[T comparable](list []T, element T) bool {
 func CheckSubnetAuthKeys(walletKeys []string, subnetAuthKeys []string, controlKeys []string, threshold uint32) error {
 	for _, walletKey := range walletKeys {
 		if slices.Contains(controlKeys, walletKey) && !slices.Contains(subnetAuthKeys, walletKey) {
-			return fmt.Errorf("wallet key %s is a subnet control key so it must be included in subnet auth keys", walletKey)
+			return fmt.Errorf("wallet key %s is a control key so it must be included in blockchain auth keys", walletKey)
```

Comment:
```suggestion
			return fmt.Errorf("wallet key %s is a control key so it must be included in auth keys", walletKey)
```

Reply:
the auth keys and valid signatures produced by them are still called "SubnetAuth", i.e. in [`ConvertSubnetToL1Tx`](https://github.com/avalanche-foundation/ACPs/blob/main/ACPs/77-reinventing-subnets/README.md#convertsubnettol1tx)